tools/webpack-core/src/utils/resolve-recurse/LICENSE.md

```diff
@@ -0,0 +1,22 @@
+  

```

Comment:
Instead of adding the License.md here, can you add a `ThirdPartyNotices.md` at the root of the package, as a peer of our LICENSE.md.  There's an example of one in core/frontend/ThirdPartyNotices.md.

It basically has the same content but bundles all of them together at the top and then links to the original source for attribution.

Reply:
Also, you may need to update the `.npmignore` to make sure the file will be published with the package.

Reply:
Changed it